angular-app/src/app/shared/components/course-form/course-form.component.html

```diff
@@ -0,0 +1,100 @@
+<form class="course-form" [formGroup]="courseForm" (ngSubmit)="onSubmit(courseForm.value)">
+
+
+    <!-- TITLE INPUT -->
+    <div class="form__content">
+        <h3 class="form__subtitle">Title</h3>
+        <div class="form__field">
+            <input type="text" 
+            class="form__input"
+            formControlName="title"
+            [ngClass]="{'form__input--invalid': 
+            (!titleField && isSubmitBtnPressed) || (!titleField && courseForm.get('title')?.touched)}">
+
+            <button type="submit" class="button__btn">Create course</button>
+        </div>
+
+        <div class="form__error-text"
+        *ngIf="(!titleField && isSubmitBtnPressed) || (!titleField && courseForm.get('title')?.touched)"
+        >Title is required</div>
+    </div>
+
+
+    <!-- DESCRIPTION INPUT -->
+    <div class="form__content">
+        <h3 class="form__subtitle">Description</h3>
+        <input type="text" 
+        class="form__input form__input--large"
+        formControlName="description"
+        [ngClass]="{'form__input--invalid': 
```

Comment:
As best practice, avoid long conditions in templates, keep conditions in .ts files. For example in function isFormValid(): boolean or getFormClass(): string